lang/pl/case_studies.php

```diff
@@ -3,9 +3,232 @@
 declare(strict_types=1);
 
 return [
-    "Project" => "Projekt",
-    "Client" => "Klient",
-    "Goal" => "Cel",
-    "Scope" => "Zakres",
-    "Technologies" => "Technologie",
+    "project_description" => "Opis projektu",
+    "challenges" => "Wyzwania",
+    "typography" => "Typografia",
+    "sitemap" => "Mapa strony",
+    "key_functionalities" => "Kluczowe funkcjonalności",
+    "architecture_schema" => "Schemat architektury systemu",
+    "color_palette" => "Paleta kolorów",
+    "primary" => "Kolor główny",
+    "secondary" => "Kolor dodatkowy",
+    "surface_container" => "Powierzchnia",
+    "text" => "Tekst",
+    "characters" => "Znaki",
+    "k1" => [
+        "title_1" => "System zarządzania",
+        "title_2" => "licencjami i dystrybucji oprogramowania",
+        "project_description" => "System do zarządzania oprogramowaniem znajdującym się na komputerach uczelnianych na podstawie zdefiniowanych reguł i harmonogramów, z uwzględnieniem licencji oprogramowania.",
+        "challenges" => [
+            "challenge_1" => [
+                "title" => "Zapewnienie bezpieczeństwa danych",
+                "description" => "lorem ipsum tutaj potrzebujemy zdanie uzupełniające",
+            ],
+            "challenge_2" => [
+                "title" => "Implementacja modelu przywilejów i ról",
+                "description" => "zapewniający właściwy rozdział uprawnień",
+            ],
+            "challenge_3" => [
+                "title" => "Kontrola i nadzór maszyn klienckich",
+                "description" => "dopuszczonych do funkcjonowania w systemie",
+            ],
+            "challenge_4" => [
+                "title" => "Uruchamianie dostarczanych aplikacji",
+                "description" => "bez konieczności ich instalacji w systemie operacyjnym",
```

Comment:
We install apps on computers :eyes: 
I can agree that user doesn't see installation wizard - he sees only progress of download and text "Instalowanie" when the app is installed